pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -200,7 +201,7 @@ func (p PermissionClaim) String() string {
 	if p.IdentityHash == "" {
 		return fmt.Sprintf("%v.%v", p.Resource, p.Group)
 	}
-	return fmt.Sprintf("%v:%v.%v", p.Resource, p.IdentityHash, p.Group)
+	return fmt.Sprintf("%v:%v.%v", p.IdentityHash, p.Resource, p.Group)
```

Comment:
odd order. In the requests we have resource:identity.

Reply:
can do that